backend/infrahub/core/schema/generated/attribute_schema.py

```diff
@@ -44,17 +45,17 @@ class GeneratedAttributeSchema(HashableModel):
     )
     regex: str | None = Field(
         default=None,
-        description="Regex uses to limit the characters allowed in for the attributes.",
+        description="Regex uses to limit the characters allowed in for the attributes. (deprecated: please use parameters.regex instead)",
         json_schema_extra={"update": "validate_constraint"},
     )
     max_length: int | None = Field(
         default=None,
-        description="Set a maximum number of characters allowed for a given attribute.",
+        description="Set a maximum number of characters allowed for a given attribute. (deprecated: please use parameters.max_length instead)",
```

Comment:
The Field class has a `deprecated` parameter, I wonder if we use that if it shows up in the output JSON schema with a warning in the yaml language server. As this would require rewriting the schema generator part I think we can tackle that in a followup PR but I think it could be a nice way to highlight the changes to users working with schema files.